src/less/button/base/button.less

```diff
@@ -79,7 +79,7 @@ a.fake-btn--primary {
 }
 
 /* show hover states only for non-disabled or non-loading state button */
-button.btn--primary:not([disabled]):not([aria-live="polite"][aria-disabled="true"]),
+button.btn--primary:not([disabled]):not([aria-disabled="true"]),
 a.fake-btn--primary[href] {
     &:focus,
     &:hover,
```

Comment:
should we drop the active here? I think maybe I should've with my last PR. Maybe could do in a separate PR if needed, but probably shouldn't be here. Other than that LGTM